moler/cmd/unix/iperf.py

```diff
@@ -0,0 +1,436 @@
+             'Transfer Raw': 27892121, 'Interval': '3.0- 4.0 sec'},
+            {'Bandwidth': '218 Mbits/sec', 'Bandwidth Raw': 228589568, 'Transfer': '26.0 MBytes',
+             'Transfer Raw': 27262976, 'Interval': '4.0- 5.0 sec'},
+            {'Bandwidth': '220 Mbits/sec', 'Bandwidth Raw': 230686720, 'Transfer': '26.2 MBytes',
+             'Transfer Raw': 27472691, 'Interval': '5.0- 6.0 sec'},
+            {'Bandwidth': '224 Mbits/sec', 'Bandwidth Raw': 234881024, 'Transfer': '26.8 MBytes',
+             'Transfer Raw': 28101836, 'Interval': '6.0- 7.0 sec'},
+            {'Bandwidth': '218 Mbits/sec', 'Bandwidth Raw': 228589568, 'Transfer': '26.0 MBytes',
+             'Transfer Raw': 27262976, 'Interval': '7.0- 8.0 sec'},
+            {'Bandwidth': '216 Mbits/sec', 'Bandwidth Raw': 226492416, 'Transfer': '25.8 MBytes',
+             'Transfer Raw': 27053260, 'Interval': '8.0- 9.0 sec'},
+            {'Bandwidth': '221 Mbits/sec', 'Bandwidth Raw': 231735296, 'Transfer': '26.4 MBytes',
+             'Transfer Raw': 27682406, 'Interval': '9.0-10.0 sec'},
+            {'Bandwidth': '222 Mbits/sec', 'Bandwidth Raw': 232783872, 'Transfer': '265 MBytes',
+             'Transfer Raw': 277872640, 'Interval': '0.0-10.0 sec'}]},
+    'INFO': ['Client connecting to 10.1.1.1, TCP port 5001', 'TCP window size: 16384 Byte (default)']
+}
+
+
+COMMAND_OUTPUT_basic_server = """
+xyz@debian:~$ iperf -u
+------------------------------------------------------------
+Server listening on UDP port 5001
+Receiving 1470 byte datagrams
+UDP buffer size: 8.00 KByte (default)
+------------------------------------------------------------
+[904] local 10.1.1.1 port 5001 connected with 10.6.2.5 port 32781
+[ ID]   Interval         Transfer        Bandwidth         Jitter        Lost/Total Datagrams
+[904]   0.0- 1.0 sec   1.17 MBytes   9.84 Mbits/sec   1.830 ms   0/ 837   (0%)
+[904]   1.0- 2.0 sec   1.18 MBytes   9.94 Mbits/sec   1.846 ms   5/ 850   (0.59%)
+[904]   2.0- 3.0 sec   1.19 MBytes   9.98 Mbits/sec   1.802 ms   2/ 851   (0.24%)
+[904]   3.0- 4.0 sec   1.19 MBytes   10.0 Mbits/sec   1.830 ms   0/ 850   (0%)
+[904]   4.0- 5.0 sec   1.19 MBytes   9.98 Mbits/sec   1.846 ms   1/ 850   (0.12%)
+[904]   5.0- 6.0 sec   1.19 MBytes   10.0 Mbits/sec   1.806 ms   0/ 851   (0%)
+[904]   6.0- 7.0 sec   1.06 MBytes   8.87 Mbits/sec   1.803 ms   1/ 755   (0.13%)
+[904]   7.0- 8.0 sec   1.19 MBytes   10.0 Mbits/sec   1.831 ms   0/ 850   (0%)
+[904]   8.0- 9.0 sec   1.19 MBytes   10.0 Mbits/sec   1.841 ms   0/ 850   (0%)
+[904]   9.0-10.0 sec   1.19 MBytes   10.0 Mbits/sec   1.801 ms   0/ 851   (0%)
+[904]   0.0-10.0 sec   11.8 MBytes   9.86 Mbits/sec   2.618 ms   9/ 8409  (0.11%)
+xyz@debian:~$"""
+
+COMMAND_KWARGS_basic_server = {
+    'options': '-u'
+}
+
+COMMAND_RESULT_basic_server = {
+    'CONNECTIONS': {
+        'local 10.1.1.1 port 5001 connected with 10.6.2.5 port 32781': [{'Bandwidth': '9.84 Mbits/sec',
```

Comment:
Almost done. Bandwith raw should be output from command(9.84 Mbits/sec), bandwith should be in basic units(10317987)

Reply:
Why? It would be inconsistent with how for example 'Transfer' is parsed.

Reply:
Transfer of course should be parsed like Bandwidth.